client/api/managers/systems.js

```diff
@@ -50,6 +50,42 @@ class SystemsManager
         await this.loading;
         return _.find(this.systems, { id: systemID });
     } // end getSystem
+
+    getStatusDisplay(desc)
+    {
+        switch (desc)
+        {
+            case 'dev':
+                return 'Early Development';
+
+            case 'beta':
+                return 'Public Beta';
+
+            case 'disabled':
+                return 'Disabled';
+
+            default:
+                return desc;
+        } // end switch
+    } // end getStatusDisplay
+
+    getStatusDescription(desc)
+    {
+        switch (desc)
+        {
+            case 'dev':
+                return 'This system is incomplete and likely to have frequent changes.';
+
+            case 'beta':
+                return 'This system is mostly complete and considered reasonably stable, but not bug free.';
+
+            case 'disabled':
+                return 'This system is disabled for normal users. USE AT YOUR OWN RISK.';
```

Comment:
I didn’t see any code that actually changed or masked the system, so I’m. It sure the description is accurate.

Reply:
It's in `/server/routes/systems/index.js` line 35.

What we do in the route is we filter out any systems you don't have permissions to see, aka: disabled ones. So the client doesn't even know these systems exist, as they aren't returned to the client.

Reply:
So reading through that code, at least a bit, it’s either see all disabled Systems, or none. For now that may be okay, but we may need a more fine-grained control in the future. For example, a new gm adds a system to get group buy-in on rpgkeeper, but it’s very rough. We don’t need his group to see all disabled systems, just the one. 

Maybe eventually make this less a permission and more a user option?